girder/girder_large_image/web_client/vue/components/HistogramEditor.vue

```diff
@@ -1,7 +1,8 @@
 <script>
-import { restRequest } from '@girder/core/rest';
-import { nextTick } from 'vue';
-import { makeDraggableSVG } from '../utils/drag';
+import {restRequest} from '@girder/core/rest';
+import {nextTick} from 'vue'; // eslint-disable-line
```

Comment:
We could also do `import Vue from 'vue';` and then use `Vue.nextTick` instead of `nextTick`, if you think that is better than the eslint disable.